analyzers/src/SonarAnalyzer.CSharp/Rules/TypeNamesShouldNotMatchNamespaces.cs

```diff
@@ -48,7 +49,7 @@ public sealed class TypeNamesShouldNotMatchNamespaces : SonarDiagnosticAnalyzer
                 "Drawing", "Dynamic", "EnterpriseServices", "Globalization", "IdentityModel",
                 "InteropServices", "IO", "JScript", "Linq", "Location", "Management", "Media",
                 "Messaging", "Microsoft", "Net", "Numerics", "Printing", "Reflection", "Resources",
-                "Runtime", "security", "server", "servicemodel", "serviceprocess", "speech",
```

Comment:
Comparison is set to `InvariantCultureIgnoreCase` so it doesn't change the meaning, but I changed it to be consistent with the other entries.

Reply:
Further: this list is outdated and should take newer namespaces into account.